### PR TITLE
Adds a bunch of feedback vars

### PR DESCRIPTION
**Pull request by psykzz**
## About The Pull Request

This mimics the current round statistics to ensure the stats are saved in the database.

## Why It's Good For The Game

Working on that statbus

## Screens
image

### CODE REVIEW

**Comment by Hughgent** [comment]
This is it. This is the thing I always wanted yet was unable to do.

File: code/controllers/subsystem/blackbox.dm [hunk 1]
```diff
@@ -174,3 +174,59 @@ SUBSYSTEM_DEF(blackbox)
 	key_type = new_key_type
 	version = new_version
 	json = list()
+
+
+/datum/controller/subsystem/blackbox/proc/ReportDeath(mob/living/L)
+	set waitfor = FALSE
+	if(sealed)
+		return
+	if(!L || !L.key || !L.mind)
```
Comment (Rohesie) [inline code/controllers/subsystem/blackbox.dm]:
```suggestion
	if(!L?.key || !L.mind)
```

**Comment by spookydonut** [comment]
Do we actually have database access?

File: code/controllers/subsystem/blackbox.dm [hunk 2]
```diff
@@ -174,3 +174,59 @@ SUBSYSTEM_DEF(blackbox)
...
 	json = list()
+
+
+/datum/controller/subsystem/blackbox/proc/ReportDeath(mob/living/L)
+	set waitfor = FALSE
+	if(sealed)
+		return
+	if(!L?.key || !L.mind)
+		return
+	var/sqlname = L.real_name
```
Comment (Rohesie) [inline code/controllers/subsystem/blackbox.dm]:
```suggestion
	var/sqlname = sanitizeSQL(L.real_name)
```
And so on. No need to define the vars before the connect check, right?

Comment (psykzz) [reply code/controllers/subsystem/blackbox.dm]: I was thinking about that, but i assume its because they want the check as late as possible?

**Pull request merged**